.gitmodules

```diff
@@ -2,10 +2,6 @@
 	path = inference-engine/thirdparty/ade
 	url = https://github.com/opencv/ade.git
 	ignore = dirty
-[submodule "inference-engine/thirdparty/mkl-dnn"]
-	path = inference-engine/thirdparty/mkl-dnn
-	url = https://github.com/openvinotoolkit/oneDNN.git
-	ignore = dirty
```

Comment:
extra change

Reply:
`install_dependencies.sh`
and
`inference-engine/thirdparty/mkl-dnn`
still incorrect